projects/plugins/jetpack/modules/videopress/class.videopress-options.php

```diff
@@ -1,11 +1,27 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName
+/**
+ * VideoPress Options
+ *
+ * @package automattic/jetpack
+ */
 
+/**
+ * VideoPress Options class.
+ */
 class VideoPress_Options {
 
-	/** @var string */
+	/**
+	 * Option name.
+	 *
+	 * @var string
```

Comment:
```suggestion
	 * @var string $option_name The 'videopress' option name
```
Building out the variable documentation with the name and a description.